client/components/auth-form.js

```diff
@@ -25,7 +25,8 @@ const AuthForm = (props) => {
         </div>
         {error && error.response && <div> {error.response.data} </div>}
       </form>
-      <a href="/auth/google">{displayName} with Google</a>
+      <a href="/auth/google">{displayName} with Google</a> <br />
```

Comment:
What is this `<br />` tag?

Reply:
It splits the line so the two Oauth links aren't side by side.

Reply:
Could also be wrapped in divs later (probably will be later)